.ci/jobs/defaults.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+##### GLOBAL METADATA
+
+- meta:
+    cluster: devops-ci
```

Comment:
Might be worth adding a view  and including the job into it  to make it easier to find the job when browsing the CI server.